src/lib/AddressUtils.js

```diff
@@ -63,9 +66,7 @@ class AddressUtils {
      * @param {string} str
      */
     static _alphabetCheck(str) {
-        const filteredStr = str.split('').filter((c) => {
-            return AddressUtils.NIMIQ_ALPHABET.includes(c);
-        }).join('');
+        const filteredStr = str.split('').filter(c => AddressUtils.NIMIQ_ALPHABET.includes(c)).join('');
```

Comment:
A classic for loop that breaks on the first "non-nimiq" char should be more performant than string>array>filter>string just to compare two strings. Of course this has nothing to do with linting, but is this a function being used often? Then we should definitely fix it.